src/node/rpc/serialization.h

```diff
@@ -9,13 +9,16 @@
 namespace ccf
 {
   DECLARE_JSON_ENUM(
-    GetSignedIndex::State,
-    {{GetSignedIndex::State::ReadingPublicLedger, "readingPublicLedger"},
-     {GetSignedIndex::State::ReadingPrivateLedger, "readingPrivateLedger"},
-     {GetSignedIndex::State::PartOfNetwork, "partOfNetwork"},
-     {GetSignedIndex::State::PartOfPublicNetwork, "partOfPublicNetwork"}})
-  DECLARE_JSON_TYPE(GetSignedIndex::Out)
-  DECLARE_JSON_REQUIRED_FIELDS(GetSignedIndex::Out, state, signed_index)
+    ccf::State,
+    {{ccf::State::uninitialized, "uninitialized"},
```

Comment:
This serialisation is now defined twice, also in `node_state.h` to format the `enum` when printing it with `fmt::format()`. Could we only define it once?

Reply:
Let me see.